### DISCUSSION TITLE
Prayer Groups

### DISCUSSION THREAD

**Discussion by prykon** [Pray4Movement/prayer-global-porch#116] [Ideas]
After we roll out users, it'd be nice if we could create Prayer Groups. A church could be a prayer group, a group of friends could be one too.

Prayer Groups should how much all the people that are in the group prayed as a sum of their individual stats. It can be useful for tracking a group's prayer efforts vs. individual prayer efforts.

Also, if a church or small group wants to pray together but can't commit to a custom lap, groups can be a good alternative option.

**Nice to haves:**
- Allow a group to have their own avatar
- Shareable link to join group
- Group stats page
- Group members page
- Display a group creation date

**Comment by squigglybob** [comment]
Thanks for writing this up @prykon 😃

**Comment by BeanMan** [comment]
来信已收到。

**Comment by okpozu155** [comment]
Good good, I'm ready.
